src/ifElse.test.js

```diff
@@ -1,11 +1,42 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
+  let firstCallback;
+  let secondCallback;
 
-  it('should ', () => {
+  beforeEach(() => {
+    firstCallback = jest.fn();
+    secondCallback = jest.fn();
+  });
+
+  it('should be declared', () => {
+    expect(ifElse).toBeInstanceOf(Function);
+  });
 
+  it(`should run first callback if condition returns 'true'`, () => {
+    const condition = jest.fn();
```

Comment:
seems like duplicate `should run first callback if condition returns 'true'` test - remove it